src/pages/ALLO_ManageAllocations.page

```diff
@@ -45,9 +45,12 @@
                         thisRowPercent.prop('disabled', true);
                     }
                 }
-                if (!isNaN($(o).val()))
-                    sumAmount += Math.round($(o).val()*100);
-               
+                var amt=$(o).val();
+                // due to how chrome handle's the global replace, have to
+                // first remove the currency symbols first and then strip out all commas.
+                if (!isNaN(amt.replace(/[\$£€]/g, '').replace(/,/g, '')))
+                    sumAmount += Math.round(amt.replace(/[\$£€]/g, '').replace(/,/g, '')*100);
```

Comment:
The only concern I have with filtering out the comma is that in some locales the comma is used in place of the decimal. Can you try to see if there is another solution that doesn't remove the comma completely?